bin/dump-dev-program-config

```diff
@@ -21,4 +16,17 @@ docker run --rm -it \
     -t versions \
     -t versions_programs \
     -t versions_questions \
-    postgres > /project/dev_programs.dump"
+    postgres -f /project/dev_programs.dump"
+
+readonly COMPOSE_CMD="docker compose \
+  -f docker-compose.yml \
+  -f docker-compose.dev.yml"
+
+${COMPOSE_CMD} \
+  up db \
+  --no-deps \
+  --wait \
+  -d
+
+${COMPOSE_CMD} exec -it -e "PGPASSWORD=example" db ${CMD}
+${COMPOSE_CMD} cp db:/project/dev_programs.dump dev_programs.dump
```

Comment:
this isn't covered by a test, so please confirm the script still works with manual testing